tuning/data/setup_dataprocessor.py

```diff
@@ -107,15 +107,21 @@ def _get_pretokenized_dataset_handlers(data_args, packing, is_eval_tokenized):
 
 
 ### Data format 2
-def _get_dataset_formatting_handlers(data_args, packing):
+def _get_dataset_formatting_handlers(data_args, packing, padding_free: str = ""):
 
     if data_args.response_template is None:
         if packing is False:
-            raise ValueError(
-                "Since dataset_text_field or data_formatter_template \
-                is provided and packing is disabled, \
-                needs a corresponding response template for masking"
-            )
+            if padding_free:
+                logger.debug(
+                    "when packing is false but padding_free is used and"
+                    + " no response template is used then its a pretrained scenario."
```

Comment:
Please chante the message here to explain the scenario properly in logs...this reads more like a comment to me.

Reply:
done